db/dcrpg/internal/addrstmts.go

```diff
@@ -56,14 +56,17 @@ const (
 									addresses.funding_tx_hash,
 									addresses.value,
 									transactions.block_height,
-									transactions.block_hash
+									block_time,
+									funding_tx_vout_index,
```

Comment:
please check with @dmigwi to be sure that the planned new address scheme will support this operation

Reply:
Sure. All those columns exists in the PR.

Reply:
This is an addresses table schema.
 https://github.com/dmigwi/dcrdata/blob/edb2c98594bbafd88ac81dc55e3e90ed22e81646/db/dcrpg/internal/addrstmts.go#L30-L40.

Reply:
It would be great if vin_vout_row_id is working now for both vins and vouts.